src/DI/WebpackExtension.php

```diff
@@ -81,6 +82,11 @@ public function loadConfiguration(): void
 			throw new ConfigurationException('You need to specify the dev server URL.');
 		}
 
+		if ($config['manifest']['mapper'] !== NULL) {
+			if (!class_exists($config['manifest']['mapper'])) {
+				throw new ConfigurationException("Non-existent classname '{$config['manifest']['mapper']}'provided for ManifestMapper.");
```

Comment:
I don't think this is necessary, the DI container will throw an exception anyway if it cannot find the class.

Reply:
Oki, I don't have much experience with writing DI extensions so any advice welcome :)
will change.